### PR TITLE
Adding Unicode arrows to player inventory

### PR DESCRIPTION
**Pull request by MrRar**
screenshot_20170325_122654

### CODE REVIEW

File: mods/creative/inventory.lua [hunk 1]
```diff
@@ -96,9 +96,9 @@ function creative.register_tab(name, title, items)
 					list[current_player;main;0,5.85;8,3;8]
 					list[detached:creative_trash;main;4,3.3;1,1;]
 					listring[]
-					button[5.4,3.2;0.8,0.9;creative_prev;<]
-					button[7.25,3.2;0.8,0.9;creative_next;>]
-					button[2.1,3.4;0.8,0.5;creative_search;?]
++					button[5.4,3.2;0.8,0.9;creative_prev;◄]
+ +					button[7.25,3.2;0.8,0.9;creative_next;►]
+ +					button[2.1,3.4;0.8,0.5;creative_search;→]
```
Comment (sfan5) [inline mods/creative/inventory.lua]: what is the `+` doing here?

**Comment by SmallJoker** [comment]
thx_

Thanks custom font, for full Unicode support \*cough*.

File: mods/creative/inventory.lua (hunk 1 above)
Comment (SmallJoker) [inline mods/creative/inventory.lua]: Diff copy fail :man_facepalming:

Comment (MrRar) [inline mods/creative/inventory.lua]: I don't know. I've removed them.

**Comment by MrRar** [comment]
@SmallJoker are you testing other arrows or my arrows?

**Comment by SmallJoker** [comment]
@MrRar Your arrows, what else. Apparently the fallback font is only used for other languages but not rarely used characters that aren't supported by some fonts.

**Comment by DS-Minetest** [comment]
I don't like the arrow for searching.
Also there would be others like , ,  or . (Not to see on github.)
:mag: ←emoticon What about a loup? letter→ 🔍
```
UTF-8: 0xF0 0x9F 0x94 0x8D
UTF-16: 0xD83D 0xDD0D

C-Oktal terminiertes UTF-8: \360\237\224\215
Dezimale XML-Entität: &#128269;
```

**Comment by kaeza** [comment]
I tried doing this in the past and it's a mess. Some people like to use custom fonts and you can't be sure the font will have those glyphs.

IMHO, it should be done with image buttons and plain white icons for better portability.

**Comment by kaeza** [comment]
The [Material Icons Library](https://material.io/icons/) contains some simple and recognizable icons, but I'm not entirely sure if the license ([Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0.txt)) is compatible.

  

Note: these examples are black to be able to see them here; you can get white versions from that page.

**Comment by kaeza** [comment]
This is how it looks with icons and a bit of tweaking:

**Comment by paramat** [comment]
:-1: For above reasons.

**Comment by SmallJoker** [comment]
@DS-Minetest PNG compression tool's fault.

I agree with kaeza and paramat here: If changed, images should be used. :-1: from my side

**Comment by paramat** [comment]
See #1666

**Comment by paramat** [comment]
Closing due to font issue, also the 'search' icom makes no sense, no one will know what it means.

**Comment by bigfoot547** [comment]
> , ,  or 

Lol! This is what I see with my font! *Replacement characters!*
screenshot from 2017-05-12 18 42 12

**Comment by DS-Minetest** [comment]
@bigfoot547 I don't remember. But using unicode chars make no sense anyway.

**Pull request closed without merging**